Vagrantfile

```diff
@@ -23,7 +23,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # Execute commands in the frontend directory
     fe.exec.commands %w(rails rake rspec bundle), directory: '/vagrant/src/api'
     fe.exec.commands 'script/start_test_backend', directory: '/vagrant/src/api'
-    fe.exec.commands 'contrib/start_development_backend', directory: '/vagrant'
+    fe.exec.commands 'start_development_backend', directory: '/vagrant/contrib/'
+    fe.exec.commands 'start_development_backend', prepend: 'sh'
```

Comment:
Why do you think you need to prepend `sh`?  The script has a shebang and is executable...